Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ignotus666 commented on _translator-files/po/de/Client-Troubleshooting.po:
```diff
@@ -25,7 +25,7 @@ msgstr "de"
 #, no-wrap
 msgid "wiki"
-msgstr "wiki"
+msgstr "Wiki"
```
This shouldn't be capitalised. I left a comment on Weblate but not sure if the translator has seen it. Correcting it in one string will correct all the other ones.

ignotus666 replied:
Done

ann0see commented on _translator-files/po/de/Command-Line-Options.po:
```diff
@@ -41,17 +42,15 @@ msgstr "Jamulus - Befehlszeilen Optionen"
+#, no-wrap
 msgid "Shared commands"
-msgstr "## Befehle für Client und Server"
+msgstr "Gemeinsame Befehle"
```
I'm not that fond of "Shared commands" as it's not understandable out of context.

ann0see commented on _translator-files/po/de/Running-a-Server.po:
```diff
@@ -94,20 +93,23 @@ msgstr ""
 #, fuzzy, no-wrap
-#| msgid "## Basic requirements"
 msgid "Requirements"
 msgstr "## Grundlegende Anforderungen"
```
I believe this is wrong (no `#` here)
```suggestion
msgstr "Grundlegende Anforderungen"
```

ann0see replied:
Fixed on Weblate

ann0see commented on _translator-files/po/de/Running-a-Server.po:
```diff
@@ -179,27 +214,44 @@ msgstr ""
+"{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - "
+"not the default `amd64` ones for use on Intel/AMD machines."
+msgstr ""
+"Lade die [neueste headless (amd64) .deb file]({{ site.download_root_link }}"
```
The link isn't translated correctly. "headless .deb file" is not German...

ann0see commented on _translator-files/po/de/Running-a-Server.po:
```diff
@@ -179,27 +214,44 @@ msgstr ""
-msgid "Make sure you have a current list of standard packages: `sudo apt update`"
-msgstr "Aktualisiere apt, um sicherzustellen, dass die Liste der Standardpakete aktuell ist: sudo apt update"
+msgid ""
+"Make sure you have a current list of standard packages: `sudo apt update`"
```
... current list of standard packages isn't correct. Dropping standard is probably best, as we don't know what a "standard" package is.

ann0see reviewed (commented):
Hmm. There are some comments.

henkdegroot commented:
@ann0see , I noticed there is a lot to translate/update on weblate. Will try to fix by 27 march on weblate.

henkdegroot commented:
@ann0see, a separate note, it seems some links are broken in the text  due to the updates made to the page. Will these be checked using a script?

ann0see commented:
I think so - we run a linkchecker before releasing the site. But it's good to fix them now - if you come across some.

henkdegroot commented:
@ann0see, they will need to be fixed in the source....it might cause some conflicts on weblate/PR. In the Running a Server page, there are server types (1, 2 and 3). In weblate, the 1, 2 and 3 have been removed and the wording is changed. I.e. 1. Registered is now Registered mode. I notice a couple of links pointing to these. I can update the translation but that is also really hard using weblate to understand all links.

ann0see commented:
No. Then don't fix them now. We should get this PR in first.

henkdegroot commented:
Checking the website locally and notice the following error:
![po4a-error](https://user-images.githubusercontent.com/13550012/228138812-a44c6c04-087a-4112-9dcb-b02a89b7322a.png)

Don't think this is caused by the translation, but not sure how best to report the error.

Maybe it is related, I noticed the following text does not show translated:
![website-translation](https://user-images.githubusercontent.com/13550012/228139459-77f49b4a-9a23-4b9f-b09c-8253cab376f8.png)

And I have checked, it is translated in the copyright.po file.

ignotus666 commented:
The po4a message is nothing to worry about, it's just pointing out a potential error. We'll have to look into why the translation of that text doesn't get picked up.

henkdegroot commented:
@ignotus666, the ASIO text in the footer is also not showing translated text. Do you want me to create a new issue for this?

henkdegroot commented:
Please advise if you want me to report/create separate issues or report issues I find on this PR.

The below is related to the macOS installation page.

English view:
![image](https://user-images.githubusercontent.com/13550012/228328384-b3efd70e-6ef2-4951-be65-9d2cd11fe48f.png)

Translated nl view:
![image](https://user-images.githubusercontent.com/13550012/228328505-daa95ef1-a9e3-44c7-87ae-562e108fd0cd.png)

As you can spot, the list is not handled correct in the translation.

ignotus666 commented:
@henkdegroot yes please, maybe open an issue for each type of problem. In the one you posted images of, it's because of the formatting of the EN text. Adding a new line after "To open Jamulus" fixes it. There isn't even a need to update the translations after correcting it - they display ok (just tested locally). Thanks for pointing this stuff out!

henkdegroot commented:
@ignotus666, I created a few issues :) and used [Next-release] in the header so they can easily be identified.

henkdegroot commented:
> We'll have to look into why the translation of that text doesn't get picked up.

Create issue #914 for this, which also contains a solution.

ignotus666 commented:
Thanks a lot!

henkdegroot commented:
> In the one you posted images of, it's because of the formatting of the EN text. Adding a new line after "To open Jamulus" fixes it. There isn't even a need to update the translations after correcting it - they display ok (just tested locally).

Not sure how you have tested or which file you updated. When I add the new line, the list is fixed but it is not showing the translation. And updating the po files, will require updating the translation as well. Think these changes need to be done after merging this PR.

ignotus666 commented:
> Not sure how you have tested or which file you updated. When I add the new line, the list is fixed but it is not showing the translation. And updating the po files, will require updating the translation as well. Think these changes need to be done after merging this PR.

I added the new line in /wiki/en/Installation-for-Macintosh.md, ran `po4a-update-templates.sh` and then `po4a-create-all-targets.sh`. Building the website locally it all looks ok and the translations are there - except in places where it's missing because I'm using next-release without the latest translation updates. The source text should never be edited in .po files.

But yes, before correcting the EN files, we should wait for this PR to go in.

ann0see commented on _translator-files/po/de/Installation-for-Windows.po:
```diff
@@ -63,10 +64,8 @@ msgstr "Updatest du? Es macht Sinn die [Konfiguration](Software-Manual#sicherung
-#| msgid "**Download and install an ASIO Driver**. Try to use the driver that your hardware manufacturer provides. If you can't find that, or you don't have an external sound card, you probably need to install ASIO4ALL. For more information scroll down to the [ASIO](#asio) section.\n"
 msgid "**Download and install an ASIO Driver**. Try to use the driver that your hardware manufacturer provides. If you can't find that, or you don't have an external sound card, you probably need to install ASIO4ALL. For more information scroll down to the [ASIO](#asio) section."
-msgstr "**Downloade und installiere einen ASIO-Treiber**. Wir empfehlen, eine Soundkarte/Interface mit einem nativen, herstellereigenen ASIO-Treiber zu verwenden. Wenn kein nativer ASIO-Treiber vorhanden ist (v.A. bei integrierten Soundkarten), brauchst du einen generischen Treiber wie ASIO4ALL. Mehr Informationen findest du im [ASIO](#asio) Abschnitt unten auf dieser Seite.\n"
+msgstr "**ASIO-Treiber herunterladen und installieren**. Versuche, den Treiber zu verwenden, den der Hersteller deiner Hardware bereitstellt. Wenn du diesen nicht finden kannst oder keine externe Soundkarte hast, musst du wahrscheinlich ASIO4ALL installieren. Weitere Informationen findest du im Abschnitt [ASIO](#asio)."
```
Should remain a sentence

ann0see replied:
Fixed on weblate

ann0see commented on _translator-files/po/de/Installation-for-Windows.po:
```diff
@@ -159,10 +156,9 @@ msgstr "Wenn das Audio nicht sofort funktioniert, stelle sicher, dass die **rich
+#, no-wrap
 msgid "How to set up ASIO4ALL inputs (Guide)"
-msgstr "### ASIO4ALL Ein- und Ausgänge einrichten (Anleitung)"
+msgstr "So richtest du ASIO4ALL-Eingänge ein (Anleitung)"
```
This might lead to broken links. Also it's a heading.

ann0see replied:
--> link checker should find this on the release. Thus resolving

ann0see commented on _translator-files/po/de/Running-a-Server.po:
```diff
@@ -94,20 +94,23 @@ msgstr ""
 #, fuzzy, no-wrap
-#| msgid "### Speed and latency"
 msgid "Speed and latency"
 msgstr "### Geschwindigkeit und Latenz"
```
To change

ann0see replied:
Fixed.

ann0see commented on _translator-files/po/de/Running-a-Server.po:
```diff
@@ -761,44 +883,51 @@ msgstr ""
+"If the Server receives a SIGUSR1 signal during a recording, it will start a "
+"new recording in a new directory. SIGUSR2 will toggle recording on/off."
+msgstr ""
+"Wenn die Aufzeichnungsfunktion mit der Befehlszeilen Option „R“ verwendet "
```
Too much translation. Already commented that on Weblate.

ann0see reviewed (approved):
Approving German

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

ann0see commented:
@ignotus666 please look over your translation again - then freel free to merge this.

ignotus666 reviewed (approved):
Spanish reviewed again. Looks fine so merging this.
